.github/workflows/continuous-integration-dotnet.yml

```diff
@@ -3,9 +3,21 @@ name: Build, analyse and test .NET code
 on:
   push:
     branches: [ main ]
+    paths:
+      - 'DfE.FindInformationAcademiesTrusts/**'
+      - 'DfE.FindInformationAcademiesTrusts.Data/**'
+      - 'DfE.FindInformationAcademiesTrusts.Data.AcademiesDb/**'
+      - '!tests/playwright/**'
```

Comment:
I think the exclusion should be after the inclusion otherwise the exclusion won't work - https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths